src/model/formats/test-data.ts

```diff
@@ -84,3 +84,86 @@ export const EXPECTED_CONTENTS: ChecklistFile = {
     copyrightInfo: 'Test copyright',
   },
 };
+
+export const EXPECTED_FOREFLIGHT_CONTENTS = {
```

Comment:
Is this because we don't (yet?) support features like indenting/centering in Foreflight files? If so, please add a TODO to add that support later (it's only natively supported on Garmin, but we can play with ways to represent the same elsewhere, as in other formats).

Reply:
Initially, I started with a separate mock because my goal was to achieve a roundtrip, and you have to start with something manageable. I generally agree that it is desirable to have the same mock for all formats.

However, now that I've finished the mapping, I'm more and more inclined to try to push you to leave it as is. The reason is that the ForeFlight format is not line or field based, and the interesting test cases are not field lengths or setting some property, but rather all kinds of missing pieces and dangerous item order. That's how I developed my mock - the ForeFlight file has all the missing things that can go missing, and I'm also testing item order and note collapsing.

We can try to merge the mocks, but the ForeFlight format doesn't support it, and I don't see a good way to introduce support for the following features:

1. Item indentation (except for attaching text items to headings and/or challenges as notes)
2. Centering of items
3. Metadata

I've tried to make the test data in 666ec05 a bit more grokable, but I'm not sure if it helps my cause.

If you still want to merge them, we can try after the metadata story is behind us, but I'd like to ask that I be able to move my groups over in addition to the existing ones, so that the edge cases I mentioned above are all properly tested and I have to do some magic to ignore unsupported stuff for ForeFlight. I'd need your help to regenerate the other files though, as I don't have access to other avionics.

Reply:
> I don't have access to other avionics

I only have access to Garmin, btw.

> more inclined to try to push you to leave it as is

I'm ok with this. We can also play more with formatting afterwards and see if we come up with something we like, no need to get it all right on this PR.